src/adt-models/AdtModels.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why is this a top level project? Isn't it just used for the Shell repository server?